### PR TITLE
created nightly rhmi install on aws clusters

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
https://issues.jboss.org/browse/INTLY-3295

### CODE REVIEW

**Comment by pawelpaszki** [comment]
successful execution can be found here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/osd-ocp4-testing-nightly-trigger/21/console

File: jobs/integr8ly/ocp4/ocp4-testing-executor.yaml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      - string:
+          name: GH_CLIENT_SECRET
+          description: "GitHub Client Secret for OAuth Apps, required for some of the walkthroughs"
+      - string:
+          name: TEST_SUITES_REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: "Repository of Integreatly test suites, YAML templates are stored there"
+      - string:
+          name: TEST_SUITES_BRANCH
+          default: 'development'
```
Comment (tremes) [inline jobs/integr8ly/ocp4/ocp4-testing-executor.yaml]: Should we go with master? Since there's plan to stop using development.

Comment (pawelpaszki) [reply jobs/integr8ly/ocp4/ocp4-testing-executor.yaml]: hmm. yes we probably should

**Comment by tremes** [comment]
There's no test execution in ocp4-testing-executor or am I missing something?

**Comment by pawelpaszki** [comment]
> There's no test execution in ocp4-testing-executor or am I missing something?

Not at this stage. It will be added once we have some tests running against OS4 clusters

File: jobs/integr8ly/ocp4/ocp4-testing-executor.yaml [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+            - '1'
+            - '2'
+            - '3'
+          description: 'Number of control plane replicas for the Openshift 4 cluster (openshift-install specific, e.g. non-pds)'
+          default: '1'
+      - choice:
+          name: TO_DO
+          description: "It specifies what stages of the pipeline will be executed. Provision and deprovision are for non-pds clusters only"
+          choices:
+            - provision + install + deprovision
```
Comment (trepel) [inline jobs/integr8ly/ocp4/ocp4-testing-executor.yaml]: Would it be useful to also have `provision + install`?

Comment (pawelpaszki) [reply jobs/integr8ly/ocp4/ocp4-testing-executor.yaml]: yes. I just put the initial array of options, which will be expanded later anyway (e.g. to include tests). Will add this option and merge. Thanks

**Pull request merged**